snowflake/pkg/infra/manager.go

```diff
@@ -488,3 +416,118 @@ func (m *Manager) Offboard(ctx context.Context) error {
 	_, err := m.run(ctx, true)
 	return err
 }
+
+func createUdfs(ctx context.Context, logger logr.Logger, databaseName string, warehouseName string, workdir string) error {
+	logger.Info("creating UDFs")
```

Comment:
Just curious about whether we have a convention on whether to capital the first letter for `logger.Info`? I see both in code.

Reply:
I prefer capitaling the first letter, let me fix that.